streaming/index.js

```diff
@@ -312,7 +297,7 @@ const startServer = async () => {
 
     if (subs[channel].length === 0) {
       log.verbose(`Subscribe ${channel}`);
-      redisSubscribeClient.subscribe(channel, onRedisMessage);
+      redisSubscribeClient.subscribe(channel)
```

Comment:
Missing semicolon, another style issue I'm not sure why it isn't caught.

Reply:
I'm also a little confused as to how these weren't caught by linting, I would've expected this to be picked up. Might be worth opening a ticket/PR reviewing the linting rules in place.